geometry_msgs/Point destination
---
RoverArm/joint_angles angles
